src/FirebaseArray.js

```diff
@@ -7,24 +7,25 @@
    *
    * Internally, the $firebase object depends on this class to provide 5 $$ methods, which it invokes
    * to notify the array whenever a change has been made at the server:
-   *    $$added - called whenever a child_added event occurs
-   *    $$updated - called whenever a child_changed event occurs
-   *    $$moved - called whenever a child_moved event occurs
-   *    $$removed - called whenever a child_removed event occurs
+   *    $$added - called whenever a child_added event occurs, returns the new record, or null to cancel
```

Comment:
It's also probably not important to get carried away here as this is all covered in the docs. But I did want some notes in here for the code hackers.